src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,19 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
```

Comment:
We now have to follow this doc https://wiki.folio.org/display/DD/Logging when creating new code. `mod-circulation` is still not fully covered with the new-style logging yet, but the new code already needs to meet this criteria. I think it would be nice to at least cover `addPrimaryServicePointNameToStaffSlipContext` method